lighthouse-cli/test/smokehouse/test-definitions/errors/error-expectations.js

```diff
@@ -46,7 +50,7 @@ const infiniteLoop = {
 const expiredSsl = {
   lhr: {
     requestedUrl: 'https://expired.badssl.com',
-    finalUrl: 'https://expired.badssl.com/',
+    finalUrl: /(expired.badssl.com|chrome-error)/,
```

Comment:
I'm not actually sure which of these is the more correct answer here, but seems like a big change regardless (especially since we use `finalUrl` for labelling the report)?

Reply:
IMO, `chrome-error://` is the more correct answer here. We could add back the redirect warning for this case if that would be useful?

Reply:
> IMO, `chrome-error://` is the more correct answer here. We could add back the redirect warning for this case if that would be useful?

I can see it either way so that WFM, but adding back the redirect warning for `chrome-error` pages definitely makes sense in that case.

Reply:
Added the redirect 👍